src/test/components/elements/Dashboard.test.tsx

```diff
@@ -32,18 +32,23 @@ describe("Dashboard", () => {
     expect(promoTile).toBeNull();
   });
 
-  it("if TEACHER user is logged in and promo item is available, the promo tile is visible, and featured news tile is not", async () => {
-    setupTest("TEACHER");
-    const promoTile = await screen.findByTestId("promo-tile");
-    const featuredNewsTile = screen.queryByTestId("featured-news-item");
-    expect(promoTile).toBeInTheDocument();
-    expect(featuredNewsTile).toBeNull();
-    const promoTitle = screen.getByText(mockPromoItem.title);
-    expect(promoTitle).toBeInTheDocument();
-  });
+  let roles: TestUserRole[] = ["TEACHER", "EVENT_LEADER", "CONTENT_EDITOR", "EVENT_MANAGER", "ADMIN"];
```

Comment:
Oops yes this was due to the fact that "ANONYMOUS" is not a UserRole so rather than having `UserRole | "ANONYMOUS"` in some tests, I used TestUserRole. However in this case it's definitely not required.